libbeat/kibana/client_test.go

```diff
@@ -32,55 +32,51 @@ import (
 
 func TestErrorJson(t *testing.T) {
 	// also common 200: {"objects":[{"id":"apm-*","type":"index-pattern","error":{"message":"[doc][index-pattern:test-*]: version conflict, document already exists (current version [1])"}}]}
-	kibanaTs := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+	kibanaTS := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusUnauthorized)
-		w.Write([]byte(`{"message": "Cannot export dashboard", "attributes":{"objects":[{"id":"test-*","type":"index-pattern","error":{"message":"action [indices:data/write/bulk[s]] is unauthorized for user [test]"}}]}}`))
+		_, err := w.Write([]byte(`{"message": "Cannot export dashboard", "attributes":{"objects":[{"id":"test-*","type":"index-pattern","error":{"message":"action [indices:data/write/bulk[s]] is unauthorized for user [test]"}}]}}`))
+		assert.NoError(t, err)
 	}))
-	defer kibanaTs.Close()
+	defer kibanaTS.Close()
+
+	assertConnection(t, kibanaTS.URL, http.StatusUnauthorized)
+}
 
+func assertConnection(t *testing.T, URL string, expectedStatusCode int) {
```

Comment:
Here you could add a call to [`t.Helper()`](https://pkg.go.dev/testing#T.Helper):
```suggestion
func assertConnection(t *testing.T, URL string, expectedStatusCode int) {
	t.Helper()
```

This helps with the error messages when the test fails.

Reply:
Done